### DISCUSSION TITLE
怎么在issue像这样子提及到另一个issue

### DISCUSSION THREAD

**Discussion by Ad-closeNN** [Meloong-Git/PCL#3362] [提问] [answered]
就像这样:
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/c9b33163-5660-4cda-a5dd-cc07413695be)

**Comment by Ad-closeNN** [comment]
哦哦会了（
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/9b7fa7b7-d6c3-4c0c-beaa-ee18e8e05635)

**Reply by hejiehao** [reply]
~~好奇怪的配色呀~~
直接把链接粘上去就行了

**Reply by Deep-Dark-Forest** [reply]
> ~好奇怪的配色呀~ 

![image](https://github.com/Hex-Dragon/PCL2/assets/131975043/ade027db-5451-46dd-adfa-63799bd3e4a5)
~我这配色更奇怪~

**Comment by Deep-Dark-Forest** [comment]
最笨的方法：
[#???]+(issue链接)

**Reply by hejiehao** [reply]
其实连 [#???] 都不需要

**Reply by hejiehao** [reply]
按 # 也可以弹出这个
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/4070b29e-7a8d-4f70-bc2f-d40d1ee3f0f1)

**Reply by hejiehao** [reply]
#114  怎么了（）

**Reply by Deep-Dark-Forest** [reply]
> 按 # 也可以弹出这个
> ![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/4070b29e-7a8d-4f70-bc2f-d40d1ee3f0f1)
> 

确实可以，也可以直接输入井号然后在后面加数字

**Reply by Deep-Dark-Forest** [reply]
> #114  怎么了（）

试一下直接用井号加数字好不好使（

**Comment by allMagicNB** [comment, marked as answer]
直接#+编号，如果后面是标点符号的话后面不用加空格，如果不是的话后面要加空格

**Reply by hejiehao** [reply]
如果是跨仓库或者是具体某个 Comment 可能还需要把整个链接粘贴上去

**Reply by allMagicNB** [reply]
`右上角三个点->Copy Link`也行

**Reply by Deep-Dark-Forest** [reply]
https://github.com/Hex-Dragon/PCL2/discussions/3362#discussioncomment-8507150

**Reply by hejiehao** [reply]
我直接引用友商
https://github.com/HMCL-dev/HMCL/discussions/2640#discussioncomment-8314700
https://github.com/orgs/PrismLauncher/discussions/695#discussioncomment-4558260

**Reply by Deep-Dark-Forest** [reply]
> 我直接引用友商
> https://github.com/HMCL-dev/HMCL/discussions/2640#discussioncomment-8314700
> https://github.com/orgs/PrismLauncher/discussions/695#discussioncomment-4558260

额？下面的链接没有正确显示

**Reply by allMagicNB** [reply]
不对啊@hejiehao 你不是这个项目的贡献者吗（

**Reply by zkitefly** [reply]
> 不对啊@hejiehao 你不是这个项目的人吗

![Screenshot_2024-02-18-21-48-48-37_df198e732186825c8df26e3c5a10d7cd.jpg](https://github.com/Hex-Dragon/PCL2/assets/64117916/bfb24156-c4fd-4381-9117-ae44e4796bef)

**Reply by hejiehao** [reply]
> > 我直接引用友商
> > [HMCL-dev/HMCL#2640 (comment)](https://github.com/HMCL-dev/HMCL/discussions/2640#discussioncomment-8314700)
> > https://github.com/orgs/PrismLauncher/discussions/695#discussioncomment-4558260
> 
> 额？下面的链接没有正确显示

不要在意这种小细节

**Reply by hejiehao** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/98140773/fa1bd831-d55b-49a8-8085-337a3e67a818)

**Comment by Ad-closeNN** [comment]
`is:open`~~都看不到这条只开了2分钟的Discussion你们怎么看的~~

**Reply by Ad-closeNN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/96520694-fbe2-496f-b000-c698bcd29ef9)
![image](https://github.com/Hex-Dragon/PCL2/assets/103355026/6e1f9aa1-2d6f-4148-8e19-170022a92b8b)

**Reply by hejiehao** [reply]
你轰炸了我的邮箱（）

**Reply by Ad-closeNN** [reply]
> 你轰炸了我的邮箱（）

不好意思啦()

**Reply by hejiehao** [reply]
NP
